mod-finance/examples/ledger.sample

```diff
@@ -9,6 +9,8 @@
   "available": 1000,
   "unavailable": 1000,
   "currency": "USD",
+  "restrictEncumbrance": false,
+  "restrictExpenditures": false,
```

Comment:
Instead of having all `false` values, maybe few could be `true` so we cover all cases